src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -833,6 +796,38 @@ PMComplexTest >> testTwoComplexNumbersWithDifferentRealPartsAreNotEqual [
  	self deny: z equals: w.
 ]
 
+{ #category : #'testing - arithmetic' }
+PMComplexTest >> testWeCanAddComplexNumbersToAnArrayOfIntegers [
```

Comment:
I don't know, but this seems like a good test to have
And I like the name
Maybe make it shorter: `testAddComplexToArrayOfIntegers`

I think this `WeCan` is never a good idea because for any test, if it is passing, then `yes we can!` :)
Haha